USING_WITH_PYRIGHT.md

```diff
@@ -4,9 +4,17 @@
 
 Pyright can also be used on the [command line](https://github.com/microsoft/pyright/blob/main/docs/command-line.md) to produce json or text output. This can be used in a variety of situations, like a [pre-commit hook](https://github.com/microsoft/pyright/blob/main/docs/ci-integration.md#running-pyright-as-a-pre-commit-hook), or a [github action](https://github.com/jakebailey/pyright-action).
 
-
 However, Pylance and Pyright don't always produce the same results. There are several reasons why this may happen.
 
+## Type checker differences

```

Comment:
Maybe this title isn't clear? Would it be clearer to say `Pyright and Pylance version differences`?

Reply:
> Pyright and Pylance version differences

That phrasing seems weird to me since Pyright versions (1.1.xxx) and Pylance versions (2023.xx.xxx) are never the same. Maybe something more verbose like one of these?

- Standalone version of Pyright differs from Pylance's version of Pyright
- Pyright version and Pylance's underlying Pyright version are different

Reply:
I like your second choice. That's what I was intending to convey.